sem-service-check-params

```diff
@@ -141,7 +141,8 @@ params-check::redis () {
 }
 
 params-check::mongodb () {
-  default_params='--net=host --rm --name mongodb -d'
+  default_params='--net=host --rm --name mongodb -d -v /var/tmp/mongodb:/home/mongo'
+  sudo mkdir -p /var/tmp/mongodb ; sudo chown -R 999:999 /var/tmp/mongodb
```

Comment:
shouldn't we use `&&` instead of `;` here?

Reply:
Also, not sure which user and group id `999` is referring to. Is that the user/group Mongo uses?

Reply:
Changed every `;` to `&&` .
999 is mogo user id in containers, changed mode to 0777 instead of owner, to be consistent with the rest of the containers.